README.md

```diff
@@ -149,9 +149,9 @@ Sun/Oracle. Sun famously didn't include ECC in a couple generations of server pa
 
 [NASA](http://www.doneyles.com/LM/Tales.html). A design flaw in the Apollo 11 rendezvous radar produced excess CPU load, causing the spacecraft computer to restart during lunar landing.
 
-[NASA](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter). Use of different units of measurement (metric vs. English) caused Mars Climate Orbiter to fail. This is basically the same issue Sweden ran into in 1628 with its ship.
+[NASA](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter). Use of different units of measurement (metric vs. English) caused Mars Climate Orbiter to fail. (This is basically the same issue Sweden ran into in 1628 with its ship.) There were also organisational and procedural failures[[ref](http://space.stackexchange.com/a/20241)] and defects in the navigation software[[ref](http://spectrum.ieee.org/aerospace/robotic-exploration/why-the-mars-probe-went-off-course)].
```

Comment:
Might I suggest dropping the parenthetical reference to the sinking of the _Vasa_? It is listed here, but much further down the file, and without anchor linking it feels like a distraction.